### PR TITLE
Turn holodeck areas into a map element

### PR DESCRIPTION
**Pull request by unid15**
- Removed holodeck areas from boxstation, test_box, defficiency and metaclub
- Created maps/misc/holodeck.dmm: 
- Maps which had their holodeck areas removed now load the above file instead

also added _load_map_elements_ list to maps that handles loading of map elements

### CODE REVIEW

**Comment by CptWad** [comment]
this is good
I can replace said holodeck file with a holodeck file with more holodeck
holodeck

File: code/modules/awaymissions/maploader/reader.dm [hunk 1]
```diff
@@ -198,6 +198,11 @@ var/global/dmm_suite/preloader/_preloader = null
 	if(_preloader && instance)
 		_preloader.load(instance)
 
+	//The areas list doesn't contain areas without objects by default
+	//We have to add it manually
+	if(!areas.Find(instance))
+		areas.Add(instance)
```
Comment (clusterfack) [inline code/modules/awaymissions/maploader/reader.dm]: addsorted

**Comment by clusterfack** [comment]
Shouldn't the holodecks be split up?

**Comment by CptWad** [comment]
How it works right now is seperate areas loading onto a base area. 
I guess it could work either way but just one map would be less clutter and less seperate map files to load

**Pull request merged**